src/main/java/rx/observables/StringObservable.java

```diff
@@ -225,16 +220,13 @@ public boolean process(byte[] next, ByteBuffer last, boolean endOfInput) {
                                 bb.put(last);
                                 bb.put(next);
                                 bb.flip();
-                            }
-                            else { // next == null
+                            } else { // next == null
```

Comment:
Please avoid whitespace and code style changes as much as possible; it makes the review more difficult. If you fix or introduce a method, you are free to code in your own style but don't change any unaffected code style.